.github/workflows/create-node-pr.yml

```diff
@@ -9,6 +9,10 @@ on:
         description: "The npm spec to create the PR from"
         required: true
         default: 'latest'
+      branch:
+        description: "The major node version to serve as the base of the PR. Should be `main` or a number like `18`, `19`, etc."
+        required: true
+        default: 'main'
```

Comment:
this repo doesn't have a `main` branch.  It'd be swell if the `HEAD` trick worked here but likely we'll want this to be `latest`.

Reply:
In this case the `base` of the PR is `nodejs/node`. This argument should be rarely used and is leftover from when we did v6 releases that needed to land in a different branch. Currently we do `v8` and `v9` to their `main` branch, but it's possibly we could do `v8` releases to a different branch in the future.

Seems like the description could use an update to be more clear. I'll make a fixup with something new unless you have a suggestion here.

Reply:
oooh this is nodejs not the cli. nope that was just my brain fog.